xcell/mappers/mapper_base.py

```diff
@@ -32,9 +39,31 @@ def _get_rotator(self, coord_default):
             rot = None
         return rot
 
-    def get_signal_map(self):
+    def _get_signal_map(self):
         raise NotImplementedError("Do not use base class")
 
+    def get_signal_map(self, **kwargs):
+        if self.signal_map is None:
+            fn = '_'.join([f'{self.map_name}_signal_map' + (f'_bin{self.zbin}'
+                                                            if self.zbin is not
+                                                            None else ''),
+                           *[f'{k}{v}' for k, v in kwargs.items()],
+                           f'coord{self.coords}',
+                           f'ns{self.nside}.fits.gz'])
+
+            print(f'Saving signal_map: {fn}')
```

Comment:
remove this?

Reply:
the print statement? I think it's nice to see the path where you're saving things (not strong feelings, though)